src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -198,7 +199,8 @@ public void updateStudentCascade(String originalEmail, StudentAttributes student
         StudentAttributes originalStudent = getStudentForEmail(student.course, originalEmail);
         updateStudentCascadeWithSubmissionAdjustmentScheduled(originalEmail, student, hasDocument);
         
-        /* finalEmail is the string to be used to represent a student's email.
+        /*
+         * finalEmail is the string to be used to represent a student's email.
          * This is because:
          *  - originalEmail cannot be used when student's email is being updated with a new valid email
```

Comment:
This is a block comment for developers to read and I think the format here is for viewers' convenience:)